databases/de-database-schema/src/main/tables/76_job_status_updates.sql

```diff
@@ -0,0 +1,48 @@
+SET search_path = public, pg_catalog;
+
+--
+-- Records job status updates
+--
+CREATE TABLE job_status_updates (
+    id uuid NOT NULL DEFAULT uuid_generate_v4(),
+
+    -- corresponds to the external_id field in the job_steps table
+    external_id character varying(64) NOT NULL,
+
+    -- the update message. needs to be freeform since we can't guarantee
+    -- the length
+    message text NOT NULL,
+
+    -- The actual status field associated with this update. again, we can't
+    -- guarantee the format from external job sites, so this ends up being
+    -- a text field
+    status text NOT NULL,
+
+    -- The ip address that the update was sent from. This should be set to the
+    -- condor node ip address or to the agave callback source ip.
+    sent_from inet NOT NULL,
+
+    -- The hostname provided in the update message.
+    sent_from_hostname text NOT NULL,
+
+    -- Send date is the date that the update was actually sent, not the date
+    -- that the record was created. Should be in milliseconds since the epoch.
+    sent_on bigint NOT NULL,
```

Comment:
Though this is sent as milliseconds since epoch, it seems like as a timestamp thing it should probably be stored as such in the DB... I'm not sure I feel strongly about it, but mostly it seems like it'd be annoying to do comparisons with actual timestamps if it's stored this way. What I can find suggests you'd have to do `'1970-01-01 00:00:00 GMT'::timestamp + 
((sent_on::bigint)::text)::interval` to convert to a timestamp, which is... rather ugly to say the least.